### PR TITLE
[bitnami/grafana] add missing resources on image renderer

### LINKED ISSUE #6923: [bitnami/grafana] image renderer does not have resources
**Which chart**: bitnami/grafana

**Describe the bug**
Image renderer does not have resources defined.

### PR DESCRIPTION
**Pull request by kuznero**
**Description of the change**

Adds resources section to image renderer.

**Benefits**

**Possible drawbacks**

**Applicable issues**

  - fixes #6923

**Additional information**

**Checklist** 

- [x] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. [bitnami/<name_of_the_chart>])

### CODE REVIEW

File: bitnami/grafana/values.yaml [hunk 1]
```diff
@@ -568,6 +568,26 @@ imageRenderer:
   ## Ref: https://kubernetes.io/docs/concepts/configuration/assign-pod-node/#affinity-and-anti-affinity
   ##
   affinity: {}
+  ## Grafana Image Renderer containers' resource requests and limits
+  ## ref: http://kubernetes.io/docs/user-guide/compute-resources/
+  ## We usually recommend not to specify default resources and to leave this as a conscious
+  ## choice for the user. This also increases chances charts run on environments with little
+  ## resources, such as Minikube. If you do want to specify resources, uncomment the following
+  ## lines, adjust them as necessary, and remove the curly braces after 'resources:'.
+  ## @param resources.limits The resources limits for Grafana containers
```
Comment (alvneiayu) [inline bitnami/grafana/values.yaml:577]: Could you modify resources.limits to imageRenderer.resources.limits, please? To fix the "Generate Chart Readme / generate-chart-readme" repository test.

File: bitnami/grafana/values.yaml [hunk 2]
```diff
@@ -568,6 +568,26 @@ imageRenderer:
...
+  ## @param resources.requests The requested resources for Grafana containers
```
Comment (alvneiayu) [inline bitnami/grafana/values.yaml:578]: Could you modify resources.requests to imageRenderer.resources.requests, please? To fix the "Generate Chart Readme / generate-chart-readme" repository test.

**Review by alvneiayu — CHANGES_REQUESTED** [review]
hi @kuznero 

First of all, thanks a lot for your PR. Just some minor changes to fix the github tests.

Thanks a lot

Álvaro

File: bitnami/grafana/values.yaml (hunk 1 above)
Comment (kuznero) [reply bitnami/grafana/values.yaml]: Fixed

File: bitnami/grafana/values.yaml (hunk 2 above)
Comment (kuznero) [reply bitnami/grafana/values.yaml]: Fixed

**Review by alvneiayu — APPROVED** [review]
LGTM

**Pull request merged**